.azure-pipelines/workbooksValidations.yaml

```diff
@@ -0,0 +1,12 @@
+jobs:
+- job: "WorkbooksValidations"
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  steps:
+  - task: Npm@1
+    displayName: 'npm install'
+    inputs:
+      verbose: false
+      command: 'install'
+  - script: 'npm run tsc && node .script/workbooksValidator.js'
```

Comment:
we have 2 options to add validation. I think both options are good. Just add this comment FYI (in case you miss it)

1. run all validation in the same script, like you did - we run over the file in one loop
2. add different script foeach validation with different display name - we will run over the files in different loop, but will see the output foreach validation under different titles\jobs (maybe it clearer for users)

Reply:
Got it. I think I will leave all the Metadata validations under one script, and when we add other workbook validations we will add a separate script file for those validations..